war/src/main/webapp/scripts/hudson-behavior.js

```diff
@@ -520,8 +520,10 @@ function makeButton(e,onclick) {
         btn.addListener("click",h);
     var be = btn.get("element");
     Element.addClassName(be,clsName);
-    if(n) // copy the name
+    if(n){ // copy the name
         be.setAttribute("name",n);
+        be.setAttribute("title",n);
```

Comment:
:bug: name attribute is html form name, this isn't likely to be what you want for a title to display, is it?